travis/travis_after_tests_success

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+import os
+from coveralls import cli as coveralls_cli
+
+if os.environ.get('TESTS', True) and not os.environ.get('LINT_CHECK'):
```

Comment:
You are right, we have "0" and "1" strings, and `TESTS` could be `0` too, so we have to check the values.